pkg/minikube/problem/err_map.go

```diff
@@ -225,6 +226,12 @@ var vmProblems = map[string]match{
 		Issues: []int{6083, 5636},
 		GOOS:   []string{"linux"},
 	},
+	"NONE_CONNTRACK": {
+		Regexp: re(`[ERROR FileExisting-conntrack]: conntrack not found in system path`),
+		Advice: "Kubernetes 1.18 and higher requires conntrack to be installed, to install it on your machine for the none driver simply run `apt-get install conntrack` as root.",
```

Comment:
don't mention apt-get install...since it could be different for centos and others. maybe add a link somewhere